lib/Sources/BluetoothAction/DiscoverDescriptors.swift

```diff
@@ -60,7 +60,7 @@ struct DiscoverDescriptorsResponse: JsMessageEncodable {
     let descriptors: [Descriptor]
 
     func toJsMessage() -> JsMessage.JsMessageResponse {
-        .body(descriptors.map { $0.uuid.uuidString.lowercased() })
+        .body(descriptors.map { $0.uuid })
```

Comment:
Changed to leverage https://github.com/JuulLabs/topaz/blob/main/lib/Sources/JsMessage/JsConvertable.swift#L53-L57